plugins/runtime/html/metrics.html

```diff
@@ -0,0 +1,13 @@
+<div class="runtime-controller"
+	ng-controller="RuntimeMetricsController">
+  <h1>Metrics</h1>
+
+	<div class="row-fluid">
+		<div class="span10">
+      <pf-table-view config="tableConfig" dt-options="tableDtOptions"
```

Comment:
- Let's use a simple HTML table here. We don't need the feature of pf-table-view and it might help with the blinking issue.
- Add a title to the last column.

Reply:
will do